ragaai_catalyst/tracers/utils/utils.py

```
@@ -53,7 +53,7 @@ def get_unique_key(input_data):
         canonical_json = normalize_string(input_data)
     else:
         # If input is neither a dictionary nor a string, raise an error
-        raise ValueError("Input must be a dictionary or a string")
+        logger.error("Input must be a dictionary or a string")
```

Ruff (0.11.9)

Undefined name `logger`

(F821)